libdino/src/service/muc_manager.vala

```diff
@@ -229,7 +229,10 @@ public class MucManager : StreamInteractionModule, Object {
             foreach (Xep.Bookmarks.Conference bookmark in conferences) {
                 Jid jid = new Jid(bookmark.jid);
                 if (bookmark.autojoin) {
-                    join(account, jid, bookmark.nick, bookmark.password);
+                    string history_since = null;
+                    Conversation? conversation = stream_interactor.get_module(ConversationManager.IDENTITY).get_conversation(jid, account);
+                    if (conversation != null) history_since = conversation.last_active.to_string();
```

Comment:
MessageStorage.get_last_message(conversation).time would be a better suited source for that information.